coral-common/src/main/java/com/linkedin/coral/common/calcite/DdlSqlValidator.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2022 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.common.calcite;
+
+import org.apache.calcite.sql.SqlNode;
+
+
+public interface DdlSqlValidator {
+
+  void validate(SqlNode ddlSqlNode);
+}
```

Comment:
Who is supposed to call this (external calling code or internally inside some Coral step)? How can we unify it with the existing pipeline for validation? @aastha25 please let me know your thoughts.